apps/dynatrace/perftest/01/dynatrace-operator2.yaml

```diff
@@ -0,0 +1,13 @@
+---
+apiVersion: helm.toolkit.fluxcd.io/v2beta1
+kind: HelmRelease
+metadata:
+  name: dynatrace-operator
+  namespace: dynatrace
+spec:
+  values:
+    classicFullStack:
+      args:
+        - --set-host-group=PERF_CFT
+        - --set-network-zone=azure.cft
+        - --set-host-id-source=ip-addresses
```

Comment:
This entry` --set-host-id-source=ip-addresses` currently breaks Dynatrace - can you review SDS setup?

More details on issue: https://one.dynatrace.com/hc/en-us/restricted?return_to=https%3A%2F%2Fone.dynatrace.com%2Fhc%2Fen-us%2Frequests%2F34601